andela/titan-dev-ah-frontend
# #163518652 Users like and dislike an article

#### What does this PR do?
Implement the functionality which allows our users to like or dislike an article.
#### Description of Task to be completed?
- Convert like and dislike icons to react components
- Create like and dislike actions creators
- Create like and dislike reducers
- Bind onClick event to like and dislike icons 
- Write the test for actions creators, reducers, components and events
- Add the like component to read article page (**On Going**)
#### How should this be manually tested?
- Clone the repo and check out the following branch ft-users-like-dislike-articles-163518652
- Run `yarn start` and navigate to /article in your browser
- Click to like icon and wait for the action to be completed to see if the article was liked.
- Click to dislike icon and wait for the action to be completed to see if the article was disliked.
- Run `yarn test` to run the unit tests
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163518652
#### Screenshots (if appropriate)
#### Remark:
-  Should wait for  `#165352853` PR to be merged to be marked as completed.
The like component needs to be integrated on the Read article page
